src/components/insurance/QuoteAndBuy.vue

```diff
@@ -23,6 +23,20 @@ onMounted(() => {
 
   console.log('state', window.history.state)
 })
+addEventListener('saveQuote', (event: any) => {
+  console.log('save quote received')
+  window.location.href = '/angular/components/login'
+})
+window.addEventListener('signup', (event: any) => {
+  console.log('sign up event received')
+
+  if (!event.detail.autoSignUp) {
+    window.location.href = '/angular/components/login'
```

Comment:
Think these should be vue

Reply:
and they have changed to /angular/insurance/components ...

Reply:
good spot